service/src/main/resources/application.yaml

```diff
@@ -209,6 +209,22 @@ fpl:
     username: 'fpl-system-update@mailnesia.com'
     password: 'Password12'
 
+fee:
+  url: 'http://fees-register-api-aat.service.core-compute-aat.internal'
+  api: '/fees-register/fees/lookup'
+  types:
+    # dummy data, to be updated
```

Comment:
As advised somewhere else, probably that would be much better to be strongly typed data.